internal/proxy/proxy_test.go

```diff
@@ -17,6 +17,12 @@ const (
 	testBearer     = "Bearer "
 )
 
+func getUserInfo(login string) *github.UserInfo {
```

Comment:
I suggest you add `testing.Helper()` at the beginning of a helper function for testing.

Also, IMO, the function name `getUserInfo()` wouldn't be so suitable here but I don't come up with a better name so I don't mind so much.

Reply:
Thanks, solved https://github.com/KeisukeYamashita/spinnaker-github-proxy/pull/7/commits/7cc7995197e76ff18c0c7794cdf0caf0f31bb2e2